protocols/system_monitor.py

```
@@ -41,18 +38,18 @@
-            'healthy': is_healthy,
-            'system_info': system_info,
-            'warnings': warnings
+            "success": is_healthy,
+            "action": "system_monitoring",
+            "healthy": is_healthy,
+            "system_info": system_info,
+            "warnings": warnings,
         }
-        
+
     except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)